server/models/dataflow.js

```diff
@@ -28,27 +28,6 @@ module.exports = (sequelize, DataTypes) => {
       }
     });
 
-    dataflow.belongsToMany(models.file, {
```

Comment:
Should the models and migration files be removed for these two tables?

Reply:
yes, I think we can remove models and migrations now, I was removing them to see if there are any errors thrown from other requests, now it looks good, we are ready for a cleanup